config/migrate_test.go

```diff
@@ -73,6 +76,11 @@ func TestMigrate(t *testing.T) {
 			err = source.SetFile(file, []byte(filesData[i]))
 			require.NoError(t, err)
 		}
+
+		return func(store *config.Store) {
```

Comment:
Should we leverage [`T.Cleanup`](https://golang.org/pkg/testing/#T.Cleanup) to simplify this further?

Reply:
Generally `t.Cleanup` comes handy but here I had to provide `config.Store` later on. If you had something different in your mind, can you elaborate a bit?

Reply:
Ah, yeah, I see the difficulty here.